cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
```

Comment:
:confounded: I think these molecules will be changing long before we fix topdocs, but I can add them if that's what the team is doing.